### PR TITLE
Add codebooks for sdg 2, 3 and 4 

### PR DESCRIPTION
**Pull request by dede95**

### CODE REVIEW

File: codebooks/codebook_sdg_2.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- Food commodity markets
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of these. More detail on the specific targets is provided below.
+
+### Key terms (non-exhaustive)
+hunger, world hunger, food, agriculture, malnutrition, farmers, fishers, food production, food producers, food access, nutrition, sufficient food, malnutrition, food systems, famines, starvation, vegetation, minerals, healthy food, vitamins, crop, crop yield, crop Production, food commodities, drought, smallholder, genetic resources, soil quality, agricultural practice, seed bank, agricultural markets 
+
+### Not to be confused with
+
+There are two Sustainable Development Goals that have aspects that are closely related to Goal 1. These are:
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md]:
```suggestion
```

File: codebooks/codebook_sdg_2.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+- 12. Responsible Consumption and Production
+  - Concerns of extraction and use of resources (environmental, material, natural)
+- 13. Climate Action
+  - Concerns about climate change on the ecosystem
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md]:
```suggestion
  - Covers broader impacts of climate change on all land based ecosystems
```

File: codebooks/codebook_sdg_2.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- 15. Life On Land
+  - Sustainable use and protection of ecosystems
+
+Remember, many projects will cover more than one Goal, but to be tagged as Goal 1 they must include at least one of the Key Concepts above. Further information on other goals can be found [here](https://unstats.un.org/sdgs/metadata/).
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md]:
```suggestion
Remember, many projects will cover more than one Goal, but to be tagged as Goal 2 they must include at least one of the Key Concepts above. Further information on other goals can be found [here](https://unstats.un.org/sdgs/metadata/).
```

File: codebooks/codebook_sdg_2.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+### Examples
+
+| Project description                                                                        | Is Goal 2? |
+|--------------------------------------------------------------------------------------------|------------|
+| Project discussing environmental effects on crop yields                        | Yes        |
+| Project discussing sustainable production of wheat                                         | Yes        |
+| Project discussing the EU's reliance on protein imports                                    | Yes        |
+| Project discussing the role of proteins and other molecular structures in cancer and ageing | No         |
+| Project discussing capacity of freshwater bodies                                      | No         |
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md]:
```suggestion
| Study measuring the physical capacity of freshwater bodies                                      | No         |
```

File: codebooks/codebook_sdg_2.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+- Maintain diverse agricultural ecosystem including diversity of seeds, plants, genetics etc.
+- Agricultural research related to food/human consumption (not biofuels, etc)
+- Investment in agricultural and rural infrastructure
+- Agricultural trade restrictions and distortions
+- Food commodity markets
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of these. More detail on the specific targets is provided below.
+
+### Key terms (non-exhaustive)
+hunger, world hunger, food, agriculture, malnutrition, farmers, fishers, food production, food producers, food access, nutrition, sufficient food, malnutrition, food systems, famines, starvation, vegetation, minerals, healthy food, vitamins, crop, crop yield, crop Production, food commodities, drought, smallholder, genetic resources, soil quality, agricultural practice, seed bank, agricultural markets
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md]:
```suggestion
hunger, world hunger, food, agriculture, malnutrition, farmers, fishers, food production, food producers, food access, nutrition, sufficient food, malnutrition, food systems, famines, starvation, vegetation, minerals, healthy food, vitamins, crop, crop yield, crop production, food commodities, drought, smallholder, genetic resources, soil quality, agricultural practice, seed bank, agricultural markets 
```

File: codebooks/codebook_sdg_2.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+- Increase agricultural productivity and incomes of small-scale food producers
+- Access to land, knowledge and technology etc. for farmers
+- Sustainable and resilient food production systems
+- Maintain diverse agricultural ecosystem including diversity of seeds, plants, genetics etc.
+- Agricultural research related to food/human consumption (not biofuels, etc)
+- Investment in agricultural and rural infrastructure
+- Agricultural trade restrictions and distortions
+- Food commodity markets
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of these. More detail on the specific targets is provided below.
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md]:
```suggestion
```

File: codebooks/codebook_sdg_3.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+- Reduce the global maternal mortality ratio
+- Reduce the number deaths and illnesses from hazardous chemicals and air, and water and soil pollution*
+- End the epidemics of diseases and premature deaths caused by such diseases
+- Prevention and treatment of substance abuse
+- Support of the research and development of vaccines and medicines
+- Universal healthcare and universal access to sexual and reproductive health-care services*
+- Access to health-care services*
+
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of these. More detail on the specific targets is provided below.
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
```

File: codebooks/codebook_sdg_3.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+* There are specific goals for consequences of consumption and production, gender and reducing inequality. Please only label projects `Yes` if they mainly relate to these areas in the context of state spending.
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
* There are specific goals for broader gender inequality and other inequalities. Please only label projects `Yes` if they concern equality in a health-specific context.

```

File: codebooks/codebook_sdg_3.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+### Key terms (non-exhaustive)
+
+health, wellbeing, life quality, health risks, prevention, disease, illness, health-related, screening, therapy, disorders, healthcare services, drug discovery, pharmaceutical, cancer, reproductive health, morality rate, welfare
+
+### Not to be confused with
+
+- 10. Reduced Inequalities
+  - Access to quality services for all not specifically related to health
+- 12. Responsible Consumption and Production
+  - Concerns the impact of chemicals and waste on not only human health but also environmental health
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
```

File: codebooks/codebook_sdg_3.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+Remember, many projects will cover more than one Goal, but to be tagged as Goal 3 they must include at least one of the Key Concepts above. Further information on other goals can be found [here](https://unstats.un.org/sdgs/metadata/).
+
+For example:
+
+| Project description                                                                        | Is Goal 3? |
+|--------------------------------------------------------------------------------------------|------------|
+| Project discussing new techniques for creating prostheses for deafness                                     | Yes        |
+| Project discussing imagining strategies on patients with intermediate risk of coronary artery disease                                    | Yes        |
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]: imaging? (instead of imagining)

File: codebooks/codebook_sdg_3.md [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+| Project discussing clot formation as a potential diagnostic tool for Alzheimer's disease                                     | Yes        |
+| Project discussing a early-warning system for agriculture-related diseases such as wheat rust diseases                                    | No         |
+| Project discussing metabolic interactions in oceanic photosymbioses                                    | No         |
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
| Project discussing metabolic interactions in oceanic ecosystems                                    | No         |

```

File: codebooks/codebook_sdg_2.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+## Labelling Instructions
+
+### Key Concepts of Goal 2
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- End hunger
+- Access to food for all people, in particular the poor and people in vulnerable situations
+- Ending all forms of malnutrition
+- Increase agricultural productivity and incomes of small-scale food producers
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md]:
```suggestion
- Agricultural productivity and incomes of small-scale food producers
```

File: codebooks/codebook_sdg_2.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+- Access to land, knowledge and technology etc. for farmers
+- Sustainable and resilient food production systems
+- Maintain diverse agricultural ecosystem including diversity of seeds, plants, genetics etc.
+- Agricultural research related to food/human consumption (not biofuels, etc)
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md]:
```suggestion
- Agricultural research related to food for human consumption (not biofuels, etc)
```

File: codebooks/codebook_sdg_3.md [hunk 14]
```diff
@@ -0,0 +1,106 @@
+# Goal 3. Good Health and Well-Being
+**Ensure healthy lives and promote well-being for all at all ages**
+
+## Labelling Instructions
+
+### Key Concepts of Goal 3
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- Improving the health of millions
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
- Improving human health

```

File: codebooks/codebook_sdg_3.md [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+**Ensure healthy lives and promote well-being for all at all ages**
+
+## Labelling Instructions
+
+### Key Concepts of Goal 3
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- Improving the health of millions
+- Increase life expectancy and reduce deaths caused by external factors such as road traffic accidents
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
- Increase life expectancy 
- Infant mortality
- Reduce deaths caused by external factors e.g.. road traffic accidents

```

File: codebooks/codebook_sdg_3.md [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+- Reduce the global maternal mortality ratio
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
- Reduce the global maternal mortality rate

```

Comment (georgerichardson) [reply codebooks/codebook_sdg_3.md]: I know this goal says 'ratio' but I think out of context 'rate' makes more sense

File: codebooks/codebook_sdg_3.md [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+- Reduce the number deaths and illnesses from hazardous chemicals and air, and water and soil pollution*
+- End the epidemics of diseases and premature deaths caused by such diseases
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
- Epidemics and related premature deaths
- Communicable and non-communicable diseases
- Mental health, well-being and suicide
```

File: codebooks/codebook_sdg_4.md [hunk 18]
```diff
@@ -0,0 +1,82 @@
+# Goal 4. Quality Education
+**Ensure inclusive and equitable quality education and promote lifelong learning opportunities for all**
+
+## Labelling Instructions
+
+### Key Concepts of Goal 4
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
```

File: codebooks/codebook_sdg_4.md [hunk 19]
```diff
@@ -0,0 +1,82 @@
+# Goal 4. Quality Education
+**Ensure inclusive and equitable quality education and promote lifelong learning opportunities for all**
+
+## Labelling Instructions
+
+### Key Concepts of Goal 4
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- Ensure all children have access to quality and free pre-primary childhood development, care and education.
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
- Quality and free, equitable and quality primary and secondary education
- Quality and free pre-primary childhood development, care and education
- Access to tertiary education and university

```

File: codebooks/codebook_sdg_4.md [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+**Ensure inclusive and equitable quality education and promote lifelong learning opportunities for all**
+
+## Labelling Instructions
+
+### Key Concepts of Goal 4
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- Ensure all children have access to quality and free pre-primary childhood development, care and education.
+- Equal access for men and women affordable vocational, technical and tertiary education
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
- Equal access for men and women to vocational, technical and tertiary education

```

File: codebooks/codebook_sdg_4.md [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+- Eliminate gender disparities in access to education and vocational training and for the vulnerable*
+- Increase in number of youth and adults who have relevant skills for employment and entrepreneurship
+- Achieve literacy and numeracy amongst youth and some adults
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
- Literacy and numeracy for youth and adults

```

File: codebooks/codebook_sdg_4.md [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+## Labelling Instructions
+
+### Key Concepts of Goal 4
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- Ensure all children have access to quality and free pre-primary childhood development, care and education.
+- Equal access for men and women affordable vocational, technical and tertiary education
+- Eliminate gender disparities in access to education and vocational training and for the vulnerable*
+- Increase in number of youth and adults who have relevant skills for employment and entrepreneurship
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
- Relevant skills for employment and entrepreneurship for youth and adults

```

File: codebooks/codebook_sdg_4.md [hunk 23]
```diff
@@ -0,0 +1,82 @@
...
+- Achieve literacy and numeracy amongst youth and some adults
+- Leaners to acquire knowledge and skills needed to promote sustainable development such as human rights, gender equality, promotion of cultural diversity, etc*
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
- Learning knowledge and skills to promote sustainable development such as human rights, gender equality, promotion of cultural diversity, etc* 

```

File: codebooks/codebook_sdg_4.md [hunk 24]
```diff
@@ -0,0 +1,82 @@
...
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- Ensure all children have access to quality and free pre-primary childhood development, care and education.
+- Equal access for men and women affordable vocational, technical and tertiary education
+- Eliminate gender disparities in access to education and vocational training and for the vulnerable*
+- Increase in number of youth and adults who have relevant skills for employment and entrepreneurship
+- Achieve literacy and numeracy amongst youth and some adults
+- Leaners to acquire knowledge and skills needed to promote sustainable development such as human rights, gender equality, promotion of cultural diversity, etc*  
+- increase the supply of qualified teachers and scholarships available to developing countries
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
- Supply of qualified teachers and scholarships available to developing countries
- Education facilities that are safe and child, disability and gender sensitive
```

File: codebooks/codebook_sdg_3.md [hunk 25]
```diff
@@ -0,0 +1,106 @@
...
+- Prevention and treatment of substance abuse
+- Support of the research and development of vaccines and medicines
+- Universal healthcare and universal access to sexual and reproductive health-care services*
+- Access to health-care services*
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md:26]:
```suggestion
- Access to health-care services*
- Universal health coverage, health insurance and access to medicines
- Development, training and retention of the health workforce
- Early warning, risk reduction and management of national/international health risks
```

File: codebooks/codebook_sdg_3.md [hunk 26]
```diff
@@ -0,0 +1,106 @@
...
+### Key Concepts of Goal 3
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- Improving the health of millions
+- Increase life expectancy and reduce deaths caused by external factors such as road traffic accidents
+- Reduce the global maternal mortality ratio
+- Reduce the number deaths and illnesses from hazardous chemicals and air, and water and soil pollution*
+- End the epidemics of diseases and premature deaths caused by such diseases
+- Prevention and treatment of substance abuse
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_3.md]:
```suggestion
- Prevention and treatment of substance abuse (incl. tobacco)

```

File: codebooks/codebook_sdg_2.md [hunk 27]
```diff
@@ -0,0 +1,89 @@
...
+### Key Concepts of Goal 2
+
+To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:
+
+- End hunger
+- Access to food for all people, in particular the poor and people in vulnerable situations
+- Ending all forms of malnutrition
+- Increase agricultural productivity and incomes of small-scale food producers
+- Access to land, knowledge and technology etc. for farmers
+- Sustainable and resilient food production systems
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_2.md:15]:
```suggestion
- Sustainable and resilient food production systems
- Climate adaptation in agriculture
- Soil quality
```

File: codebooks/codebook_sdg_4.md [hunk 28]
```diff
@@ -0,0 +1,82 @@
...
+
+### Key terms (non-exhaustive)
+education, quality, leaning, school, university, college, academic, vocational, training, education system, literacy, curriculum, STEM, ability, e-book, books, creativity, open learning, student, teacher training, numeracy, access to education
+
+### Not to be confused with
+
+- 5. Gender Equality
+  - Concerns gender equality in various areas, not just education (specifically on reproductive health)
+- 8. Decent Work and Economic Growth
+  - Focus on types of work conditions and improving working environments
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
```

File: codebooks/codebook_sdg_4.md [hunk 29]
```diff
@@ -0,0 +1,82 @@
...
+
+* There are specific goals for gender, reducing inequalities and gender equality. Please only label projects `Yes` if they mainly relate to these areas in the context of state spending.
+
+### Key terms (non-exhaustive)
+education, quality, leaning, school, university, college, academic, vocational, training, education system, literacy, curriculum, STEM, ability, e-book, books, creativity, open learning, student, teacher training, numeracy, access to education
+
+### Not to be confused with
+
+- 5. Gender Equality
+  - Concerns gender equality in various areas, not just education (specifically on reproductive health)
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
  - Concerns broader gender equality

```

File: codebooks/codebook_sdg_4.md [hunk 30]
```diff
@@ -0,0 +1,82 @@
...
+
+### Examples
+
+| Project description                                                                        | Is Goal 4? |
+|--------------------------------------------------------------------------------------------|------------|
+| Project discussing the European infrastructure for open learning                    | Yes        |
+| Project discussing the creation of a game which aims to develop and deliver a curriculum in soft skills                    | Yes        |
+| Project discussing a collaboration ecosystem enabling EU creative SMEs to exchange multi-media content and create multi-plot, interactive apps for children                   | Yes        |
+| Project discussing building capacity for design enabled innovation in urban environments                    | No         |
+| Project discussing improving the knowledge of organised crime                    | No         |
```
Comment (georgerichardson) [inline codebooks/codebook_sdg_4.md]:
```suggestion
| Development of non-education related games for children                    | No         |

```

**Review by georgerichardson — CHANGES_REQUESTED** [review]

File: codebooks/codebook_sdg_3.md (hunk 10 above)
Comment (dede95) [reply codebooks/codebook_sdg_3.md]:
```suggestion
| Project discussing imaging strategies on patients with intermediate risk of coronary artery disease                                    | Yes        |

```

**Pull request merged**